### DISCUSSION TITLE
How I can use Pixel Perfect Resolution?

### DISCUSSION THREAD

**Discussion by deceani** [Fannovel16/comfyui_controlnet_aux#442] [Q&A]
![image](https://github.com/user-attachments/assets/4f3240bc-0bd3-4aef-b90a-aca5eeff400e)
How I can use Pixel Perfect Resolution?
I am trying to use in control net, or just for resizing, but I can not find to what I can connect that input. So can someone help me understand me how it works and how I can use it in the workflow? it has that int output, but not sure to what other nodes in comfyui i can connect it.
Thanks

**Comment by PilockHulmes** [comment]
Right click the ControlNet Preprocessor node, click `Convert widget to input` and then click `convert resolution to input`.

![image](https://github.com/user-attachments/assets/7fcbeb72-c1b5-44e8-bd97-509f23fad2ce)

After that, the resolution will become an input and you can connect it.

![image](https://github.com/user-attachments/assets/ca022173-5350-472e-a82b-b1c3e8e7dc4d)
